pkg/api/v1/helper/helpers.go

```diff
@@ -433,6 +433,28 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) *v1.PersistentVolumeMode {
```

Comment:
IMO, it should return just the value. Is there a reason why it returns pointer?

Reply:
I also have similar method in 'pkg/volume/util/volumehelper/volumehelper.go'.
Which place is better to define them?

https://github.com/kubernetes/kubernetes/pull/51494/files/cb22f35a87ad60357070ca9e3c791bc6aa5eb2c8#diff-8bd9a94e772ec0bbd381b01aaf274bceR166

Reply:
API helpers is better, all API consumers have access to it.

Reply:
Thank you for the confrimation.